pkg/volume/csi/BUILD

```diff
@@ -12,6 +12,7 @@ go_library(
         "csi_plugin.go",
         "csi_util.go",
         "expander.go",
+        "shared_informer_hook.go",
```

Comment:
I don't think this is the right place for this type. Should this be tackled separately (is this appropriate for sig apimachinery)?